pkg/services/ccam/ccam.go

```diff
@@ -65,16 +68,49 @@ type Client struct {
 	Cluster *v1.Cluster
 }
 
+// InvestigateCCAM performs a full investigation a cluster's credentials. It will return an aws client or an error after trying to jump into support role.
+// It will also place the cluster into limited support or remove the cluster from limited support if appropriate.
+func InvestigateCCAM(awsClient aws.Client, ocmClient OcmClient, pdClient PdClient, cluster *v1.Cluster) (aws.Client, error) {
+	// Try to get cloud credentials
+	arClient := assumerole.Client{
+		Service: assumerole.Provider{
+			AwsClient: awsClient,
+			OcmClient: ocmClient,
+		},
+	}
+
+	cssJumprole, ok := os.LookupEnv("CAD_AWS_CSS_JUMPROLE")
+	if !ok {
+		return aws.Client{}, fmt.Errorf("CAD_AWS_CSS_JUMPROLE is missing")
+	}
+
+	supportRole, ok := os.LookupEnv("CAD_AWS_SUPPORT_JUMPROLE")
```

Comment:
Nice implementation, I've one small question
are there default values for these parameters? If so, it might be nice adding it or mentioning in the comments